### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by cournape**
Basic docker workflow for hondana:
1. a base image used by the other ones
2. a build image to build wheels
3. a deployment image based on 1. that install wheels from 2.

@sjagoe for the deployment container, it seems that running `CMD ["supervisord"]` without the `--no-daemon` does not start supervisor. As is, it runs as expected w/ the `-n` option, but then the container run is attached to the tty

### CODE REVIEW

File: run.docker [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+		-o APT::Install-Recommends=false -o APT::Install-Suggests=false \
+		nginx supervisor && \
+	apt-get clean && \
+	unlink /etc/nginx/sites-enabled/default && \
+	sed -e "s|HONDANA_SOURCES|/srv/hondana-git|g" -i /etc/nginx/sites-enabled/hondana && \
+	sed -e "s|HONDANA_STORE|/srv/store|g" -i /etc/nginx/sites-enabled/hondana && \
+	/srv/env/bin/pip install --no-index -f /srv/hondana-git/wheelhouse hondana gunicorn && \
+	/srv/env/bin/python \
+		/srv/hondana-git/scripts/generate_conf.py \
+		/srv/hondana-git/config.yaml.j2 /srv/hondana-git/config.yaml
```
Comment (sjagoe) [inline run.docker]: All containers started from this image will share the same secret key.

Comment (sjagoe) [inline run.docker]: Best practice suggests that all configuration should be provided from outside the container, not bundled in the image.

Comment (cournape) [inline run.docker]: should one just mount a volume containing the configuration ? Or is there a smarter way ?

Comment (sjagoe) [inline run.docker]: Yes, I would use volumes. Environment variables are often proposed in peoples' best practice suggestions, but I am not such a fan.

If you prefer, an alternative is to deploy a config template and have an `ENTRYPOINT` that is a script to template that from env vars. See the official postgres container for an entrypoint example.

**Comment by sjagoe** [comment]
LGTM

**Pull request merged**